docs/how-to-guides/integrating-autoware/creating-maps/creating-vector-map/lanelet2/index.md

```diff
@@ -0,0 +1,124 @@
+# Creating a Lanelet
+
+At this page, we will explain how to create a simple lanelet on your pointcloud map.
```

Comment:
Maybe you can add link to LIOSAM for people who see this page first.

Reply:
@mitsudome-r :+1: I added.